gramps/plugins/webreport/narrativeweb.py

```diff
@@ -1493,7 +1493,8 @@ def set_mtime(tarinfo):
                 try:
                     shutil.copyfile(from_fname, dest)
                     os.utime(dest, (mtime, mtime))
-                except:
+                except Exception as exception: # pylint: disable=broad-except
```

Comment:
Although we don't have a policy regarding this, it is generally not a good idea to disable pylint warnings.  In this case the exceptions that we should probably be catching here are `IOError` and `OSError`.  It is useful to see these warnings when running pylint.  Someone may wish to improve the code at a later date.

It is also useful to put pylint improvements in a separate commit.  This makes the other changes easier to read.  However, if it not easy to separate them out, don't bother to spend time doing so in this PR.